syncd/Syncd.cpp

```diff
@@ -917,6 +917,10 @@ sai_status_t Syncd::processBulkCreateEntry(
                 entries[it].vr_id = m_translator->translateVidToRid(entries[it].vr_id);
             }
 
+            static PerformanceIntervalTimer timer("Syncd::processBulkCreateEntry(route_entry) CREATE");
```

Comment:
>static [](http://example.com/codeflow?start=12&length=6)

Better to use class static member. There could be multiple Syncd instances, right?

Reply:
per process, no, i will fix that on the road, so far here are many performance counters instantiated like here